src/App.tsx

```diff
@@ -15,48 +17,108 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+export enum SortType {
+  Alphabet,
+  Length,
+  Default,
+}
+
+function getSortedGoods(
+  goods: string[],
+  sortField: SortType,
+  isReversed: boolean,
+): string[] {
+  const sortedGoodsCopy = [...goods];
+
+  if (sortField === SortType.Alphabet) {
+    sortedGoodsCopy.sort((good1, good2) => {
+      return good1.localeCompare(good2);
+    });
+  }
+
+  if (sortField === SortType.Length) {
+    sortedGoodsCopy.sort((good1, good2) => {
+      return good1.length - good2.length;
+    });
+  }
+
+  if (isReversed) {
+    sortedGoodsCopy.reverse();
+  }
+
+  return sortedGoodsCopy;
+}
+
 export const App: React.FC = () => {
+  const [sortField, setSortField] = useState<SortType>(SortType.Default);
+  const [isReversed, setIsReversed] = useState(false);
+  const sortedGoods = getSortedGoods(goodsFromServer, sortField, isReversed);
+
   return (
     <div className="section content">
       <div className="buttons">
         <button
           type="button"
-          className="button is-info is-light"
+          className={cn(
+            'button',
+            'is-info',
```

Comment:
```suggestion
            'button is-info',
```

can be simplify, check all places